backend/src/routes/api/modelRegistryRoleBindings/modelRegistryRolebindingsUtils.ts

```diff
@@ -27,12 +27,19 @@ export const createModelRegistryRoleBinding = async (
   rbRequest: V1RoleBinding,
   mrNamespace: string,
```

Comment:
why are you passing around this value? isn't it practically free from the config?

Reply:
This is a holdover from when we used to rely on getting the namespace from the client. You're right, we should probably not pass it around... although for this release I'm hesitant to refactor it because the helper to get that namespace can throw errors and we'll need to double check the error handling around these utils. I'll include it in my tech debt follow up issues.